.github/workflows/dotnet.yml

```diff
@@ -12,14 +12,16 @@ jobs:
     runs-on: windows-latest
     steps:
       - name: 🤘 checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           submodules: recursive
           fetch-depth: 0
       - name: ⚙ dotnet
-        uses: actions/setup-dotnet@v1
+        uses: actions/setup-dotnet@v4
         with:
           dotnet-version: '9.0.x'
+      - name: Add private NuGet source
+        run: dotnet nuget add source https://nuget.pkg.github.com/metaform/index.json --name metaform --username ${{ github.repository_owner }} --password ${{ secrets.GITHUB_TOKEN }} --store-password-in-clear-text
```

Comment:
nit: use multi-line commands for readability

Reply:
Fixed.